addons/mail/views/res_users_views.xml

```diff
@@ -35,12 +35,14 @@
                     <field name="signature" position="attributes">
                         <attribute name="widget">html_mail</attribute>
                     </field>
-                    <field name="signature" position="after">
-                        <field name="out_of_office_from" widget="daterange"
-                               options="{'end_date_field': 'out_of_office_to'}"/>
-                        <field name="out_of_office_to" invisible="1"/>  <!-- otherwise it is visible, smort -->
-                        <field name="out_of_office_message"/>
-                    </field>
+                    <group name="calendar_preferences" position="inside">
+                        <label for="out_of_office_to" string="Out-of-office" />
+                        <div id="full_date" class="o_row">
```

Comment:
From what I grepped, I don't see any real usefulness to having this ID on this DIV. Is there a technical reason?